website/docs/reference/operators/query/logical-operators/logical-operators/2-insert-dataset.request.js

```diff
@@ -0,0 +1,41 @@
+db.runCommand({
+  insert: 'books',
+  documents: [
```

Comment:
Are we using the same set of data for createIndexes? :thinking: Maybe we should create a fixture if that's the case

Reply:
The createIndexes doc doesn't have an insert, commands are only for creating indexes

Reply:
Let me rephrase, I've see similar data inserted like this. It probably wasn't for createIndexes command but for other documentation. Is it worth creating a fixture that contains subset of books collection. You've mentioned that you don't want to use existing books fixture because it contains field like `vector`. I think we should create a fixture that can be used for majority of data.

Reply:
While I understand the importance of a standard fixture, there will always be conmands or operation examples that favor some document fields more than others. And if we have to have an overload of fields to cover all use cases, then across our docs, it won't be beginner friendly. I believe we should have a balance between standard and simplicity.

And example like this is simple and easy to understand for a `findOne` command:

```
db.books.insertMany([
  { title: "1984", author: "George Orwell" }
])
```

But something with 8 fields and nested documents to cover multiple scenarios introduces unnecessary complexity.

Reply:
What we agreed on was to work on each command independently so as to avoid this kind of complexity, as long as its tested with CTS.

Reply:
Yes. With each set of commands having their own variation of fields, we have flexibility and can always cover a lot of basis and scenarios, as long as they remain tested and validated with CTS.

Reply:
Thanks for the explanation :hugs: